dev/io.openliberty.microprofile.telemetry.internal.common/src/io/openliberty/microprofile/telemetry/internal/common/info/OpenTelemetryInfoFactoryImpl.java

```diff
@@ -133,7 +126,8 @@ public OpenTelemetryInfo getOpenTelemetryInfo(ApplicationMetaData metaData) {
             if (atomicRef == null) {
                 //If this is triggered by internal code that isn't supposed to call ApplicationStateListener.applicationStarting() don't throw an error
                 String j2EEName = metaData.getJ2EEName().toString();
-                if (j2EEName.startsWith("io.openliberty") || j2EEName.startsWith("com.ibm.ws")) {
+                if (j2EEName.startsWith("io.openliberty") || j2EEName.startsWith("com.ibm.ws")
+                    || j2EEName.startsWith("arquillian-liberty-support")) {
```

Comment:
This change is ok but, as mentioned on the issue, it would be good to see if there's a way to identify whether an "application" is really a liberty feature.

It might be the case that if the application is not one we've seen starting, then we can assume it's a liberty feature.